packages/ckeditor5-watchdog/docs/features/watchdog.md

```diff
@@ -41,11 +41,9 @@ Then, change your `ClassicEditor.create()` call to `watchdog.create()` as follow
 
 ```js
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
-import { EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
+import { ClassicEditor, EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
```

Comment:
```suggestion

```

Reply:
Those imports are no longer needed - everything is imported from the main open-source package.